File: src/dlp/ast/expressions/RealLiteral.ts

import {Expression} from "./Expression";
import {AbstractLocatable} from "../AbstractLocatable";

export class RealLiteral extends AbstractLocatable implements Expression {

    value: number

    constructor( value: number, line: number, column: number){
        super(line, column);
        this.value = value;
    }
}

File: src/ast/expressions/RealLiteral.java
package ast.expressions;

import ast.AbstractLocatable;

public class RealLiteral extends AbstractLocatable implements Expression {

    private double value;

    public RealLiteral(double value, int line, int column) {
        super(line, column);
        this.value = value;
    }

    public double getValue() {
        return value;
    }
}
